### PR TITLE
chore: have flake8 check the entire project

### PR DESCRIPTION
**Pull request by JohnVillalovos**
Have flake8 run at the top-level of the projects instead of just the
gitlab directory.

### CODE REVIEW

File: tools/functional/api/test_issues.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def test_issue_notes(issue):
 
 
 def test_issue_labels(project, issue):
-    label = project.labels.create({"name": "label2", "color": "#aabbcc"})
+    _ = project.labels.create({"name": "label2", "color": "#aabbcc"})
```
Comment (max-wittig) [inline tools/functional/api/test_issues.py:42]:
```suggestion
project.labels.create({"name": "label2", "color": "#aabbcc"})
```

File: tools/functional/api/test_merge_requests.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def test_merge_requests(project):
     )
 
     source_branch = "branch1"
-    branch = project.branches.create({"branch": source_branch, "ref": "master"})
+    _ = project.branches.create({"branch": source_branch, "ref": "master"})
```
Comment (max-wittig) [inline tools/functional/api/test_merge_requests.py:17]:
```suggestion
    project.branches.create({"branch": source_branch, "ref": "master"})
```

File: tools/functional/api/test_releases.py [hunk 3]
```diff
@@ -29,7 +29,7 @@ def test_delete_project_release(project, release):
 
 
 def test_create_project_release_links(project, release):
-    link = release.links.create(link_data)
+    _ = release.links.create(link_data)
```
Comment (max-wittig) [inline tools/functional/api/test_releases.py:32]:
```suggestion
    release.links.create(link_data)
```

File: tools/functional/api/test_repository.py [hunk 4]
```diff
@@ -74,15 +74,15 @@ def test_create_commit(project):
 def test_create_commit_status(project):
     commit = project.commits.list()[0]
     size = len(commit.statuses.list())
-    status = commit.statuses.create({"state": "success", "sha": commit.id})
+    _ = commit.statuses.create({"state": "success", "sha": commit.id})
```
Comment (max-wittig) [inline tools/functional/api/test_repository.py:77]:
```suggestion
    commit.statuses.create({"state": "success", "sha": commit.id})
```

File: tools/functional/api/test_repository.py [hunk 5]
```diff
@@ -74,15 +74,15 @@ def test_create_commit(project):
...
     assert len(commit.statuses.list()) == size + 1
 
 
 def test_commit_signature(project):
     commit = project.commits.list()[0]
 
     with pytest.raises(gitlab.GitlabGetError) as e:
-        signature = commit.signature()
+        _ = commit.signature()
```
Comment (max-wittig) [inline tools/functional/api/test_repository.py:85]:
```suggestion
        commit.signature()
```

**Review by max-wittig — CHANGES_REQUESTED** [review]
Nice catch!

Do we need to assign variables that we're not using to `_`?

**Comment by JohnVillalovos** [comment]
> 
> 
> Nice catch!
> 
> Do we need to assign variables that we're not using to `_`?

No. The only reason I had for doing that was to make it clear that the call returns something.

I was undecided if I should just delete the assignment or use `_`. So I went with `_` 😀

But I will re-do it and delete the assignments.

**Comment by max-wittig** [comment]
If we want to check, if it's returning something we should assert instead. But I think this is already checked somewhere else.

You can also apply the suggestions and I squash after, if you like 😃

File: tools/functional/api/test_issues.py (hunk 1 above)
Comment (JohnVillalovos) [reply tools/functional/api/test_issues.py]: Done

File: tools/functional/api/test_releases.py (hunk 3 above)
Comment (JohnVillalovos) [reply tools/functional/api/test_releases.py]: Done

File: tools/functional/api/test_repository.py (hunk 5 above)
Comment (JohnVillalovos) [reply tools/functional/api/test_repository.py]: Done

File: tools/functional/api/test_repository.py (hunk 4 above)
Comment (JohnVillalovos) [reply tools/functional/api/test_repository.py]: Done

File: tools/functional/api/test_merge_requests.py (hunk 2 above)
Comment (JohnVillalovos) [reply tools/functional/api/test_merge_requests.py]: Done

**Comment by JohnVillalovos** [comment]
@max-wittig  Updated. I think I have changed everything as requested.

**Review by max-wittig — APPROVED** [review]

**Pull request merged**